src/epgsql_sock.erl

```diff
@@ -426,11 +426,11 @@ notify(State = #state{queue = Q}, Notice) ->
     end,
     State.
 
-notify_async(State = #state{async = Pid}, Msg) ->
-    case is_pid(Pid) of
-        true  -> Pid ! {epgsql, self(), Msg};
-        false -> false
-    end,
+notify_async(State = #state{async = Pid}, Msg) when is_pid(Pid); is_atom(Pid) ->
+    Pid ! {epgsql, self(), Msg},
```

Comment:
It even will try to send messages to `undefined` and fail of course (if there is no process registered under `undefined` atom)